torch/fx/experimental/symbolic_shapes.py

```diff
@@ -299,6 +299,14 @@ def guard_scalar(a):
     else:
         raise AssertionError(f"unrecognized scalar {a}")
 
+def _constrain_symbol_range(shape_env, s: sympy.Symbol, min: int, max: int):
```

Comment:
nit, we already have "experimental" in the module path, no need for underscore here.
```suggestion
def constrain_symbol_range(shape_env, s: sympy.Symbol, min: int, max: int):
```

Reply:
Eh, if we know this is going to be private, why not keep the underscore? One day symbolic shapes will no longer be experimental, and we don't have to do additional cleanup then...

Reply:
Anyway, don't have to block on this comment, idc so much... :)

Reply:
I guess most of the functions in this file is meant to be private... yet they don't got underscore, so my point is mostly about consistency... but yeah don't feel blocked on this.